docs/developers/l2/integration-tests.md

```diff
@@ -0,0 +1,62 @@
+# Running integration tests
+
+In this section, we will explain how to run integration tests for ethrex L2 with the objective of validating the correct functioning of our stack in our releases. For this, we will use ethrex as a local L2 dev node.
+
+## Prerequisites
```

Comment:
This should mention docker is required. I just ran everything without starting docker and it wasn't clear what was failing 🤡